ui/values/localizable/en.go

```diff
@@ -206,9 +206,10 @@ const EN = `
 "errPassEmpty" = "Password can not be empty"
 "estimatedSize" = "Estimated size"
 "estimatedTime" = "Estimated time"
-"exchange" = "Exchange"
+"exchange" = "Centralized Exchange"
```

Comment:
It's clearer to use `centralizedExchange` as the key for this. `exchange` is reserved for "Exchange",  I think.